Makefile

```diff
@@ -6,14 +6,15 @@
 VERSION_SUFFIX ?= -dev
 PROD_VERSION ?= 4.4.0${VERSION_SUFFIX}
 BUILDER_GOLANG_VERSION ?= 1.22
+TARGETARCH ?= amd64
 BUILDER_3RDPARTY_VERSION ?= $(shell echo $(PROD_VERSION) | cut -d. -f1,2)
 BUILD_DATE:=$(shell date +%Y%m%d)
 IMG_NAME ?= cluster-api-virtual-controller
 # IMG_URL ?= gcr.io/spectro-images-public/release/cluster-api-virtual/
 IMG_URL ?= gcr.io/spectro-common-dev/${USER}/cluster-api-virtual
 IMG_TAG ?= v0.1.3-spectro-${BUILD_DATE}
 IMG ?= $(IMG_URL)/$(IMG_NAME):$(IMG_TAG)
-BUILD_ARGS = --build-arg CRYPTO_LIB=${FIPS_ENABLE} --build-arg BUILDER_GOLANG_VERSION=${BUILDER_GOLANG_VERSION} --build-arg BUILDER_3RDPARTY_VERSION=${BUILDER_3RDPARTY_VERSION}
+BUILD_ARGS = --build-arg CRYPTO_LIB=${FIPS_ENABLE} --build-arg BUILDER_GOLANG_VERSION=${BUILDER_GOLANG_VERSION} --build-arg BUILDER_3RDPARTY_VERSION=${BUILDER_3RDPARTY_VERSION} --platform linux/${TARGETARCH}
```

Comment:
Sorry Ahmed - the `--platform` arg is actually not a build arg. See https://github.com/spectrocloud/palette-cli/blob/main/Makefile#L174 for an example.